src/BlockTypes/SingleProductDetails.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+
+/**
+ * SingleProductDetails class.
+ */
+class SingleProductDetails extends AbstractBlock {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'single-product-details';
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content Block content.
+	 * @param WP_Block $block Block instance.
+	 *
+	 * @return string Rendered block output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		$tabs = $this->render_tabs();
+
+		$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes );
```

Comment:
This method is useful if Global Styles are applied, but this block doesn't support them. It is fine remove this.

Furthermore, we should add this code and attach the class name to the main wrapper

```php
		$classname          = $attributes['className'] ?? '';
```
The user can defined the custom classes here:
![image](https://user-images.githubusercontent.com/4463174/215817469-f762f6b8-7ed5-4007-a693-52369ef5d551.png)

Reply:
Ohh good to know, thank you for sharing!